test/models/mnist/mnist_deep_simplified.py

```diff
@@ -0,0 +1,258 @@
+# Copyright 2015 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""A simplified deep MNIST classifier using convolutional layers.
+This script has the following changes when compared to mnist_deep.py:
+1. no dropout layer (which disables the rng op)
+2. no truncated normal initialzation(which disables the while op)
+
+See extensive documentation at
+https://www.tensorflow.org/get_started/mnist/pros
+"""
+# Disable linter warnings to maintain consistency with tutorial.
+# pylint: disable=invalid-name
+# pylint: disable=g-bad-import-order
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import argparse
+import sys
+import tempfile
+import getpass
+import time
+
+from tensorflow.examples.tutorials.mnist import input_data
+
+import tensorflow as tf
+import ngraph_bridge
+
+seed = 1
+tf.random.set_random_seed(seed)
+
```

Comment:
It is the same mnist deep_simplified.py that we have in examples/mnist. Added some changes to make the script deterministic so that every run is same and the accuracy and loss printed in the logs can be compared. 

Changes:
1. Setting random seed